src/neo/Ledger/MemoryPool.cs

```diff
@@ -62,6 +62,7 @@ public class MemoryPool : IReadOnlyCollection<Transaction>
 
```

Comment:
Not only price of opcode, but also price of interop services should be included.